docs/docs/03-tools/05-context.md

```diff
@@ -84,7 +84,7 @@ Continuing with the above example, this is how you can use it in a script:
 ```yaml
 # my_context_with_arg.gpt
 context: ./context_with_arg.gpt with ${search} as search_tool
-Args: search: Search tool to use
+args: search: Search tool to use
```

Comment:
Only for consistency within this page

Reply:
I'm probably going to go through and capitalize everything (I think that's the more canonical way now) as I work on my docs revamp, so this is fine either way for now.